.storybook/regression-test/main.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
   stories: ['../../stories/**/*.regression-test.story.jsx'],
-  addons: ['@storybook/addon-backgrounds'],
+  addons: [
+    '@storybook/addon-backgrounds',
+    '@storybook/addon-viewport/register',
```

Comment:
The addon using @storybook/addon-viewport is a dependency of the 'essentials' package, was it intentional to use a dependency of a dependency in this case? Wouldn't it be better to use the 'essentials' package specifying which plugins to use similar to the documentation?